netket/graph/common_lattices.py

```diff
@@ -112,11 +116,13 @@ def Grid(
         pbc = [pbc] * ndim
     if color_edges:
         kwargs["custom_edges"] = [(0, 0, vec) for vec in np.eye(ndim)]
+    if point_group is None:
+        point_group = _grid_point_group(extent, pbc, color_edges)
     return Lattice(
         basis_vectors=np.eye(ndim),
         extent=extent,
         pbc=pbc,
-        point_group=lambda: _grid_point_group(extent, pbc, color_edges),
+        point_group=point_group,
```

Comment:
before you were setting it to a lambda function, now it's a concrete object. was it wrong before?

Reply:
I put the lambda expression back. I think we did this because these groups can be arbitrarily large for high enough dimensions. It's not an issue to worry about in practice, but let's be on the safe side. (In `Lattice` these functions are only unwrapped in methods that explicitly use the symmetry groups, so it might save setup time for people working on 6D hypercubic lattices.)

Reply:
great, thanks!